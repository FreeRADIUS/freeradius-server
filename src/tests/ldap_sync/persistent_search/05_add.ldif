#
# ARGV: -x -H ${PERSISTENT_SEARCH_TEST_SERVER} -D "cn=manager" -w "secret123"
#
dn: cn=bosses,ou=groups,dc=example,dc=com
changeType: add
cn: bosses
objectClass: groupOfNames
objectClass: top
member: uid=bob,ou=people,dc=example,dc=com
member: uid=john,ou=people,dc=example,dc=com

