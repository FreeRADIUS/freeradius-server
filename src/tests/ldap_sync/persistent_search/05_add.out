Add ou=groups,dc=example,dc=com cn=bosses,ou=groups,dc=example,dc=com  
Group member uid=bob,ou=people,dc=example,dc=com
Group member uid=john,ou=people,dc=example,dc=com
